packages/core/src/messagingProtocols/mls/E2EIdentityService/E2EIServiceExternal.ts

```diff
@@ -155,6 +155,7 @@ export class E2EIServiceExternal extends TypedEventEmitter<Events> {
           clientId: id.client,
           qualifiedUserId: userId,
           credentialType: CredentialType.Basic,
+          x509Identity: undefined,
```

Comment:
can we add a comment to explain why `undefined` is passed here?

Reply:
Basic devices dont have a x509Identity